openedx/core/djangoapps/appsembler/sites/management/commands/custom_domains_list.py

```diff
@@ -1,15 +1,14 @@
 import json
 
+from openedx.core.djangoapps.appsembler.sites.models import AlternativeDomain
+
 from django.core.management import BaseCommand
-from django.db import connections
 
 
 class Command(BaseCommand):
     help = "Outputs the list of custom domains that will be used to generate let's encrypt certs"
 
     def handle(self, *args, **options):
-        cursor = connections['tiers'].cursor()
-        cursor.execute("SELECT custom_domain FROM organizations_microsite WHERE custom_domain != '';")
-        rows = cursor.fetchall()
-        domains = [{'domains': row} for row in rows]
+        altertive_domains = AlternativeDomain.objects.values_list('site__domain', flat=True)
```

Comment:
Should we filter by `get_active_sites`? Can we?


```suggestion
        altertive_domains = AlternativeDomain.objects.filter(
            site__in=get_active_sites(),
        ).values_list('site__domain', flat=True)
```

As long as the Ansible playbook uses `www-data` this change should be okay.

```yaml
- name: List custom domains
  become: www-data
  shell: ./manage lms custom_domains_list 
```